packages/documentation/src/stories/components/tooltip/tooltip.stories.ts

```diff
@@ -62,11 +62,17 @@ const meta: MetaComponent = {
     placement: {
       name: 'Placement',
     },
-    arrow: {
-      name: 'Arrow',
+    animation: {
```

Comment:
Why did you override the animation arg?
It would be better to use the definition derived from the JSDoc of the component.